src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressPane.cs

```diff
@@ -26,7 +26,8 @@ class ProgressPane
         internal
         ProgressPane(ConsoleHostUserInterface ui)
         {
-            if (ui == null) throw new ArgumentNullException(nameof(ui));
+            if (ui == null)
+                throw new ArgumentNullException(nameof(ui));
```

Comment:
Can't tell in this editor if it lines up right, but please add braces

```suggestion
               {
                   throw new ArgumentNullException(nameof(ui));
               }
```